packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/types.ts

```diff
@@ -1,7 +1,27 @@
-import { PaymentValue, RemoteDataType } from 'core/types'
+import { CoinTypeEnum, PaymentValue, RemoteDataType } from 'core/types'
+
+export type AccountType = {
+  address: string
+  agent: {
+    account: string | null
+    accountType: string | null
+    address: string | null
+    code: string | null
+    country: string | null
+    name: string | null
+    recipient: string | null
+    recipientAddress: string | null
+    routingNumber: string | null
+    swiftCode: string | null
+  }
+  currency: CoinTypeEnum
+  id: string
+  state: 'PENDING' | 'ACTIVE'
+}
```

Comment:
There is already a type for this called `BeneficiaryType`

Reply:
1